src/node.h

```diff
@@ -418,9 +429,7 @@ extern "C" NODE_EXTERN void node_module_register(void* mod);
       priv,                                                           \
       NULL                                                            \
     };                                                                \
-    NODE_C_CTOR(_register_ ## modname) {                              \
-      node_module_register(&_module);                                 \
-    }                                                                 \
+    NODE_C_CTOR(_register_ ## modname)                                \                                  \
```

Comment:
This is causing a problem building on Linux.
../src/async-wrap.cc:190:1: error: stray ▒\▒ in program
NODE_MODULE_CONTEXT_AWARE_BUILTIN(async_wrap, node::Initialize)
^
Maybe add an if MVS ... else ... here.